### PR TITLE
Fix build vs2019

### PR DESCRIPTION
**Pull request by Blackvals**
Changes to do proper build with VS2019 without losing any conan package along the way

### CODE REVIEW

File: ci/pipelines/Build.jenkinsfile [hunk 1]
```diff
@@ -103,7 +91,7 @@ pipeline
 	{
 		always
 		{
-			junit allowEmptyResults: true, testResults: "build*/HashUtilsTest.xml"
+			junit allowEmptyResults: true, testResults: 'build*/test_reports/*.xml'
```
Comment (Tanaka12) [inline ci/pipelines/Build.jenkinsfile:94]: Wouldn't be easier to save the test output on its repsective build folders `${buildFolder}/bin/${config}` and gather 'build*/**/*.xml ?

File: .codacy.yml
Comment (Tanaka12) [inline .codacy.yml:1]: Won't this break the static analysis step?

**Review by Tanaka12 — APPROVED** [review]
A few doubts asked, but looks pretty good

File: .codacy.yml
Comment (Blackvals) [reply .codacy.yml:1]: Until we invest time to properly set codacy or anything else, I prefer not having it

File: ci/pipelines/Build.jenkinsfile (hunk 1 above)
Comment (Blackvals) [reply ci/pipelines/Build.jenkinsfile:94]: It is a matter of choice. The idea behind test_reports folder is to not mix build products with execution products, but works both ways.

Comment (Tanaka12) [reply ci/pipelines/Build.jenkinsfile:94]: Makes sense, that's good

**Pull request merged**